### DISCUSSION TITLE
Getting no mesh of the object, only the floor

### DISCUSSION THREAD

**Discussion by IrunLF** [alicevision/Meshroom#2224] [Q&A] [answered]
Hi guys, I tried to make my first test with a small 3d printed boat, but the final mesh result only shows the floor and nothing of the boat, just like if the software completely avoided it. I don't know much about photogrammetry so any help is welcomed! Is it because the color (blue), bad light, etc? I leave 3 pics which ilustrate the problem perfectly. :)

<img width="257" alt="2" src="https://github.com/alicevision/Meshroom/assets/120263510/31fc7d60-33b1-45f3-89c0-48866fb65d50">
<img width="593" alt="3" src="https://github.com/alicevision/Meshroom/assets/120263510/7f9b19f7-2dc2-476c-933a-2a36c1c0b564">
<img width="256" alt="1" src="https://github.com/alicevision/Meshroom/assets/120263510/c253bb76-dcd8-4126-849e-ac9fb672de6b">

**Comment by natowi** [comment]
`Is it because (...) bad light, etc?` Yes. You can find some tutorials here: https://github.com/alicevision/Meshroom/wiki/Tutorials 

https://journalists.org/resources/a-guide-to-photogrammetry-photography/

https://www.3dscanstore.com/blog/3d-scanning-reflective-objects

**Comment by IrunLF** [comment]
> `Is it because (...) bad light, etc?` Yes. You can find some tutorials here: https://github.com/alicevision/Meshroom/wiki/Tutorials
> 
> https://journalists.org/resources/a-guide-to-photogrammetry-photography/
> 
> https://www.3dscanstore.com/blog/3d-scanning-reflective-objects

Is it possible to achieve a decent result with a mobile phone camera or should I surrender with it? I'm a bit surprised since the program did not even catch a single triangle of the boat surface, completely avoid it.

**Reply by natowi** [reply, marked as answer]
The issue is the surface of the blue boat. It has a single colour, is featureless and slightly reflective. Also your boat is really small. Each image should cover more boat than environment.
Click on the three dots in the image viewer to show detected features. You should see markers all over the area around the boat but far less on the boat itself. This points to issues with your object/capturing technique.
![grafik](https://github.com/alicevision/Meshroom/assets/8138537/0dd9d8d2-c42b-4f56-8119-904a29edf3e8)


If you are just getting into photogrammetry, start with something easier, like a fist sized stone with rough surface.
Your camera seems to be decent, so make yourself familiar with the basics from the links I posted. 
Having bad results is in most cases based on unsuitable objects and not following capturing best practices.

**Reply by IrunLF** [reply]
Thanks for that valuable information, I guess that red dots are the captured areas which later became on the mesh and blue dots are just ignored areas because of no coincidence or other reasons. Am I right?
<img width="319" alt="image" src="https://github.com/alicevision/Meshroom/assets/120263510/b5c83784-ce6d-4f95-a082-1e3dee144375">
